website/docs/d/entity.html.markdown

```diff
@@ -103,7 +103,8 @@ In addition to all arguments above, the following attributes are exported:
 ## Additional Examples
 
 -> If the entities are not found please try again without providing the `types` field.
-### An example of querying OTEL entities
+
+### Query for an OTEL entity
```

Comment:
You can add `type = "SERVICE"` to the Otel one. `filters=%28domain%20%3D%20%27EXT%27%20AND%20type%20%3D%20%27SERVICE%27%29`